ci/Jenkinsfiles/build.groovy

```diff
@@ -65,6 +65,8 @@ pipeline {
     AWS_REGION = 'us-east-1'
     AWS_BUCKET = "nuxeo-platform-coldstorage"
     CURRENT_NAMESPACE = nxK8s.getCurrentNamespace()
+    NPMJS_TOKEN = nxK8s.getSecretData(namespace: '${CURRENT_NAMESPACE}', name: 'nexus', key: 'npmjs\\.token')
+    NPM_REGISTRY_DOMAIN = "${NPM_REGISTRY}".substring('https:'.length())
```

Comment:
Would have been more robust to do
```suggestion
    NPM_REGISTRY_DOMAIN = "${NPM_REGISTRY}".replaceFirst(~'(?i)^https?', '')
```
but I got a 
```
 org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: Scripts not permitted to use staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods replaceFirst java.lang.String java.util.regex.Pattern java.lang.String
```
should we touch https://github.com/nuxeo/platform-ci/blob/master/charts/jenkins/values.yaml.gotmpl#L73 ?

Reply:
not needed anymore